xbmc/addons/kodi-addon-dev-kit/include/kodi/addon-instance/pvr/General.h

```diff
@@ -0,0 +1,455 @@
+/*
+ *  Copyright (C) 2005-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "../../AddonBase.h"
+#include "../../c-api/addon-instance/pvr/pvr_general.h"
+
+//¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯¯
+// "C++" Definitions group 1 - General PVR
+#ifdef __cplusplus
+
+namespace kodi
+{
+namespace addon
+{
+
+//==============================================================================
+/// @defgroup cpp_kodi_addon_pvr_Defs_PVRCapabilities_PVRRecordingLifetimeValue class PVRRecordingLifetimeValue
+/// @ingroup cpp_kodi_addon_pvr_Defs_PVRCapabilities
+/// @brief **PVR add-on timer type value**\n
+/// Representation of a timer event related value.
+///
+//@{
+class PVRRecordingLifetimeValue
```

Comment:
May I ask why this class does not follow the design pattern to wrap it's C counterpart (deriving it from public CStructHdl<>)?